demo/src/App.js

```diff
@@ -21,20 +24,33 @@ export default class App extends PureComponent {
     Object.keys(source.channels).forEach((channel, i) => {
       sliderValues[channel] = [0, 20000];
       colorValues[channel] = colorOptions[i];
+      channelsOn[channel] = true;
       sliders[channel] = withStyles({
         root: {
           color: `rgb(${colorOptions[i]})`
         }
       })(Slider);
+      checkboxes[channel] = withStyles({
+        root: {
+          color: `rgb(${colorOptions[i]})`,
+          '&$checked': {
+            color: colorOptions[i]
```

Comment:
Could you explain this in person? why do the two `colors` gets different kinds of values?

Reply:
One is for the box, and the other is for the checkmark itself.